projects/iot-cloud/software/lora2mqtt/src/main.py

```diff
@@ -0,0 +1,147 @@
+import json
+import logging
+import os
+import sys
+import atexit
+
+import paho.mqtt.publish as publish
+import requests
+from apscheduler.schedulers.background import BackgroundScheduler
+from flask import Flask
+from flask import jsonify
+from flask import request
+from flatten_json import flatten
+
+from utils.exceptions import RestException
+from utils.functions import resolve_mapping
+
+application = Flask(__name__)
+
+CONFIG_URL = os.environ.get('CONFIG_URL',
+                            'https://gist.githubusercontent.com/dorinelfilip/6743182de240959b2a723fcf232b8496/raw/'
```

Comment:
I'm sorry, what?

Reply:
That is a gist where I keep the config. If you have a better idea, fell free to propose / implement it.

Reply:
Serve the default `config.json` from the image. This URL is random and you do something to it the image will start failing without any possibility to be fixed.

Reply:
If you do not configure a URL for the configuration, it is `bafta charlie` anyway.